install_config/configuring_local.adoc

```diff
@@ -137,7 +156,7 @@ To allow the provisioner pod to delete content on local volumes created by any p
 $ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml
 ----
 
-. Instantiate the template by specifying values for `configmap`, `account`, and `provisioner_image` parameters:
+. Instantiate the template by specifying values for the `CONFIGMAP`, `ACCOUNT`, and `PROVISIONER_IMAGE` parameters:
```

Comment:
s/the `CONFIGMAP`, `ACCOUNT`, and `PROVISIONER_IMAGE`/the `CONFIGMAP`, `SERVICE_ACCOUNT`, `NAMESPACE`, and `PROVISIONER_IMAGE`/